docs/visualize/lens.asciidoc

```diff
@@ -78,19 +76,30 @@ still allows you to make the change.
 [[customize-operation]]
 ==== Change the aggregation and labels
 
-Lens allows some customizations of the data for each visualization.
+For each visualization, Lens allows some customizations of the data.
 
 . Click *Drop a field here* or the field name in the column.
 
-. Change the options that appear depending on the type of field.
+. Change the options that appear. Options vary depending on the type of field.
++
+[role="screenshot"]
+image::images/lens_aggregation_labels.png[Quick function options]
 
 [float]
 [[layers]]
 ==== Add layers and indices
 
-Bar, line, and area charts allow you to visualize multiple data layers and indices so that you can compare and analyze data from multiple sources.
+X and y charts allow you to visualize multiple data layers and indices so that you can compare and analyze data from multiple sources.
```

Comment:
I am not sure if 'X and y charts' is clear enough. The name 'XY Chart' is reserved for scatter chart so we cannot use it either. But still this is very close to 'X and y charts'. 

I don't have a good idea though except for leaving it as it was before. 
(still, not native speaker opinion, I might be wrong)

Reply:
In the Visualize docs refresh, we are renaming these three chart types "XY charts", so I'm changing here for consistency. I agree that XY charts is a bit vague, but I was told this a common term. I'm open to other suggestions!

Reply:
Is there a reason not to list the separate chart types? If we're going to group them, they are often grouped together as "XY charts," but not as "X and Y charts".

Reply:
These are all really good points. I'm going to take Marta's original advice and list them all out. I'll do the same in the Visualize docs refresh.